ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology-mpls.md

```diff
@@ -0,0 +1,189 @@
+# Fabric Topology Variables for mpls Design
+
+The fabric topology variables define the connectivity between the various node types, as well as override the default switch properties.
+
+- The mpls design supports any fabric topology variables already supported by l3ls-evpn, barring the exceptions outlined in this document.
```

Comment:
Please use MPLS everywhere except when talking about the value of the `design` variable, where it must be lower case.